shared/test/test_redis_property_bag.rb

```diff
@@ -0,0 +1,107 @@
+# Unit tests for the RedisPropertyBag.
+# This uses a fake Redis service by default, but can be configured to use a
+# a real instance on localhost by setting the USE_REAL_REDIS environment variable;
+# e.g.
+
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/helpers/redis_property_bag', __FILE__
+
+class RedisPropertyBagTest < Minitest::Unit::TestCase
+
+  def test_property_bags
+    redis = create_redis()
+
+    # A uniquifier to ensure that repeated runs do not collide on property bag
+    # names when running against real Redis.
+    uniquifier = Random.rand()
```

Comment:
Two things here:

1. I was a little concerned about this test having a random chance of failure/not entirely cleaning itself up, except that it's only an issue if you run against real Redis, so no change needed.
2. "Uniquifier" is a new term to me, so I had to look it up.  It looks like the way you're using it here doesn't quite mean the same thing it does in MS SQL Server (the first several search results for that term).  Where did you get this word?